### ISSUE TITLE
Implement file delete (rm) and directory listing (ls) functions

### ISSUE THREAD

**Issue by askvictor** [micropython/webrepl#12]
I know these can be performed over via the REPL, but so can the get_file and put_file commands; it would be nice to have the basic filesystem primitives running over the same channel.

**Comment by pfalcon** [comment]
Right, that's how webrepl protocol should evolve. There should be planning for the order of operations to be added, to minimize upfront effort/maximize outcome, however. And the next one to add is GET_VER, to control new versions of the protocol: https://github.com/micropython/micropython/commit/e15fb33e109983ced35f14c674e0aa1d47dbc357. 

And here's how community can help: integrate check for GET_VER into webrepl.html .

**Comment by gkzsolt** [comment]
Hi,
is there any news on this, is it on the roadmap at least? I use WEBREPL for my uPython projects over the years, and love it, being very nice and useful. However, now I have to dump all files on one of my boards, and I have to disconnect it and open the box to plug a USB in, just to get a directory listing.
Thank you for your work!
Best,

**Comment by gigawatts** [comment]
Waiting for this as well, but for now, here is my hack

Insert this into boot.py
```
from os import listdir as ls
ls = ls()
```

now you can just run `ls` at a repl prompt to get a list of files
